lti-tool-provider/src/main/java/sikt/lti/tp/aws/ApiGatewayLambdaLaunchHandler.java

```diff
@@ -1,16 +1,16 @@
 package sikt.lti.tp.aws;
 
-import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
 import java.net.URI;
 import java.util.Set;
+import nva.commons.apigateway.RequestInfo;
```

Comment:
I guess you can remove dependency to was-events in gradle with this change?

Reply:
removed, not in use longer